ironic-deployment/README.md

```diff
@@ -1,8 +1,36 @@
 # Kustomizations for Ironic
 
 This folder contains kustomizations for Ironic. They are mainly used
-through the [deploy.sh](../tools/deploy.sh) script, which takes care of
-generating the necessary config for basic-auth and TLS.
+traditionally been used through the [deploy.sh](../tools/deploy.sh) script,
+which takes care of generating the necessary config for basic-auth and TLS.
+
+Experimentally, instead of `deploy.sh`, you can use the new golang-based
+[deploy-cli](../hack/tools/deploy-cli) library,
+which, at the moment, handles everything `deploy.sh` does. You can either:
+
+- Run the package with `go run`:
+
+```shell
+cd $BMO_ROOT/hack/tools/deploy-cli
```

Comment:
Where is BMO_ROOT defined? I think it would be better to just that in BMO repository, cd `hack/tools/...`